tests/conftest.py

```diff
@@ -123,6 +123,13 @@ def update_metadata_callback(req):
     "DELETE::api/v1/samples/761bc54b97f64980": {},
     "GET::api/v1/samples/public": {},
     "GET::api/v1/projects/public": {},
+    "GET::api/v1/analyses/public": [{
```

Comment:
@cindyzwang This is weird mock data in that it's a _job_ vs. an analysis. Can you embed this as a `job` within an analysis here? Otherwise I worry we'll get bit by this test data later.